css/estilos-movies.css

```diff
@@ -0,0 +1,152 @@
+@import url(flexboxgrid.min.css);
+
+.seccion-dos{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    /*background-color: #38bb6a;*/
+    margin-top: 15px;
+    margin-bottom: 25px;
+    margin-right: 5px;
+    margin-left: 5px;
+    align-items: flex-start;
+}
+.seccion1movies{
+    float: left;
+    width: 100%;
+    color: #370617;
+    margin: 10px 0 0 45px;
+    padding: 0;
+    font-size: 20px;
+    text-align: center;
+    font-family: 'Mulish', sans-serif;
+}.listapeliculas{
+    display: flex;
+    justify-content: end;
+}
+.listapeliculas{
+    background-color: #fff;
+    color: #370617;
+    border: 1px solid #370617;
+    letter-spacing: 2px;
+    text-transform: unset;
+    cursor: pointer;
+    padding: 15px;
+    width: 95%;
+    margin: 5px 25px 0 0;
+    font-size: 18px;
+    text-align: center;
+    text-decoration: none;
+    font-family: 'Mulish', sans-serif;
+}
+
+.listapeliculas:hover{
+    background-color: #012a4a;
+    color: #fff;
+    text-decoration: underline;  
+}
+
+.seccion2movies{
+    width: 30%;
+    float: right;
+    margin: 0 0 0 0;
+}
+.lista{
+    width: 20%;
+    float:right;
+}
+#seccion3movies{
+    width: 100%;
+    justify-items: center;
+}
+.seccion3{
+    width: 30%;
+    color: #370617;
+    padding: 10px;
+    text-align: center;
+    float: left;
+    margin: 15px 15px 10px 15px;
+    font-family: 'Mulish', sans-serif;
+    font-size: 15px;
+}
+
+.imagenseccion3{
+    width: 255px;
+    height: 260px;
+}
+
+.seccion3categorias-etiquetas{
+    width: 35%;
+    float: right;
+    margin: 0 0 25px 0;
+}
+
+aside{
+    margin: 0 0 25px 0;
+    float: center;
+    color: #370617;
+    font-size: 14px;
+    font-family: 'Mulish', sans-serif;
+}
+
+.container-1{
+    width: 20%;
+    vertical-align: middle;
+    white-space: nowrap;
+    position: relative;
+    margin: 95px 0;
+}
+
+.container-1 input#search{
+    height: 3.5em;
+    background: #f48c06;
+    border: none;
+    font-size: 10pt;
+    float: center;
+    color: white;
+    padding-left: 20px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    border-radius: 5px;
+}
+
+.container-1 .icon{
+    position: absolute;
+    top: 50%;
+    margin-left: 17px;
+    margin-top: 17px;
+    z-index: 1;
+    color: #4f5b66;
+}
+
+.container-1 input#search:hover, .container-1 input#search:focus, .container-1 input#search:active{
+    outline:none;
+    background: #ffba08;
+}
+
+.titulo{
+    font-size: 26px;
+    font-weight: bold;
+    font-family: 'Mulish', sans-serif;
+}
+
+.botonetiquetas{
+    background-color: #f48c06;
+    color: #fff;
+    border: none;
+    letter-spacing: 2px;
+    text-transform: unset;
+    cursor: pointer;
+    padding: 15px;
+    margin-top: 10px;
+    font-size: 15px;
+    display: inline-block;
+    text-decoration: none;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;  

```

Comment:
Declare font family and fallbacks in root elements, for example, the `button` element. Notice here that the intent of the font fallbacks is as follows: If the browser doesn't find 'Lucida Sans' will use 'Lucida Sans Regular' and so on, is this your intention? I would recommend changing this property to: `'Lucida Sans', Geneva, Verdana, sans-serif`